hello-world/sjtolley/README.md

```diff
@@ -0,0 +1,14 @@
+# PREPROCESSED C HELLO WORLD
+Includes a basic Makefile to build the program. Tested on Linux Mint.
+
+The program reports the first few Fibonacci numbers and then prints `hello, world.`
```

Comment:
Every time I've run this, "hello, world." is printed *before* the first 3 Fibonacci numbers :)